script.rb

```diff
@@ -454055,8 +454055,8 @@ def raw_apply(snr)
 s.ins 0xc3, 4, 68, byte(2), 120
 s.ins 0xc4, 4, byte(1), 68
 s.ins 0x83, byte(0), 36
-s.ins 0x86, ushort(5951), byte(1), byte(1), s.layout('@r雷鳴と共に、……瓶より解放された淡い光は、まるで蝶の群れのようにふわりと飛び回る。')
-s.ins 0x86, ushort(5952), byte(1), byte(1), s.layout('@rこれほどの嵐の中に、それをまるで感じさせずにふわりふわりと飛ぶ光景は、@kとても不思議なものだった。')
+s.ins 0x86, ushort(5951), byte(1), byte(1), s.layout('@rIn time with the thunder... the faint glimmer released from the bottle began to fly about like a swarm of butterflies.')
+s.ins 0x86, ushort(5952), byte(1), byte(1), s.layout('@rThe sight of them flying about in a storm@k was a curious thing.')
```

Comment:
Put an ellipsis before `@k` or remove it altogether so that it doesn't hang in the middle of a sentence.